dev/CommonStyles/ToggleSwitch_themeresources.xaml

```diff
@@ -505,7 +505,7 @@
                                 RadiusY="10"
                                 Fill="{ThemeResource ToggleSwitchFillOff}"
                                 Stroke="{ThemeResource ToggleSwitchStrokeOff}"
-                                StrokeThickness="2" />
+                                StrokeThickness="1" />
```

Comment:
I think we should create resource key for this too?

Reply:
Fixed here 669a1e8